examples/features/README.md

```diff
@@ -26,3 +26,4 @@ To run any feature example follow steps for [Basic NSM setup](../basic)
 - [Exclude prefixes client](./exclude-prefixes-client)
 - [Select forwarder by capability](./select-forwarder)
 - [Policy based routing](./policy-based-routing)
+- [Mutually awared NSEs](./mutually-awared-nses)
```

Comment:
I's probably better to say "Mutualy aware", not "awared". Since 'aware' is an adjective, not a verb.